.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
 install: "pip install -r requirements.txt"
 
 before_script:
+  - if [ "$(git diff --name-only $TRAVIS_COMMIT_RANGE | grep -v -e 'md$' -e 'rst$')" == "" ]; then return 1; fi
```

Comment:
also, doesn't this exit with a non-zero exit code? Wouldn't this fail the build?